app/routes/github.js

```diff
@@ -28,30 +28,30 @@ export default Route.extend(AuthenticatedRouteMixin, {
     let stateValidator = get(this, 'githubState');
 
     if (stateValidator.validate(state)) {
-      return this._sendRequest(code);
+      return this._sendRequest(code, state);
```

Comment:
I added this to the API and it works. GitHub does some state validation itself, apparently. At least that's how I read the docs. Can you correct me if I'm wrong on this?